packages/stark-ui/assets/styles/_colors.scss

```diff
@@ -13,9 +13,9 @@ $mat-light-theme-foreground: $foregrounds;
   }
 }
 
-$stark-color-theme: mat-light-theme(
+$theme: mat-light-theme(
```

Comment:
Shouldn't be the name of this "stark-theme" instead of just "theme". That way it is clear it is something from "Stark"..

Reply:
All the `$stark-something` are maps that the end user can defined. On our side we declare a map call `$something` which will be merged with `$stark-something`. I think we should keep this convention.
In this example $theme is only a local var that we use to instanciate the theming, it won't be used anywhere else so the naming is not really important.